code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -534,10 +535,13 @@ GLOBAL_VAR(medibot_unique_id_gen)
 			var/obj/item/clothing/CH = H.head
 			if (CS.clothing_flags & CH.clothing_flags & THICKMATERIAL)
 				return FALSE // Skip over them if they have no exposed flesh.
+		if(H.is_bleeding()&&!H.is_bandaged())
+			return TRUE
```

Comment:
```suggestion
		if(H.is_bleeding())
			return TRUE
```

Is bleeding checks to see if you are bleeding, is bandaged checks if you have bandaged regardless of if you are still bleeding. These are now independant and you only want to check for bleeding here, since it now returns false if you aren't bleeding.

Reply:
the second check is necessary incase the patient is bandaged, but the bandage isn't good enough to completely stop the bleeding

without the !H.is_bandaged() the medbot can enter a near-infinite loop of bandaging someone

Reply:
There will be no infinite loop, bandages stack now and can be repeatedly applied until bleeding stops